hack/build-antrea-windows-all.sh

```diff
@@ -102,9 +103,11 @@ docker build \
        -t antrea/base-windows:$WIN_BUILD_TAG \
        --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
        --build-arg GO_VERSION=$GO_VERSION \
-       --build-arg NANOSERVER_VERSION=$NANOSERVER_VERSION .
+       --build-arg NANOSERVER_VERSION=$NANOSERVER_VERSION \
+       --build-arg WIN_OVS_VERSION=$WIN_OVS_VERSION .
 cd -
 
+
```

Comment:
should be reverted

Reply:
you mean cd -  ? it was not added

Reply:
the comment is commenting the extra empty line.